pkg/sql/logictest/testdata/logic_test/privilege_builtins

```diff
@@ -933,9 +940,10 @@ SELECT has_table_privilege('seq', 'SELECT'),
        has_table_privilege('seq', 'REFERENCES'),
        has_table_privilege('seq', 'TRIGGER'),
        has_table_privilege('seq', 'SELECT, INSERT, UPDATE'),
-       has_table_privilege('seq', 'SELECT, TRUNCATE')
+       has_table_privilege('seq', 'SELECT, TRUNCATE'),
+       has_table_privilege('seq', 'RULE')
 ----
-true  true  true  true  true  true  true  true  true
+true  true  true  true  true  true  true  true  true  true
```

Comment:
i'm still thinking about this -- some of these return true for RULE because the user has ALL privileges. i'm wondering if we should special-case RULE in the has_table_privilege implementation so it just always returns false.

Reply:
cc @otan ?

Reply:
i changed it to always return false